srcCpp/Infrastructure_pro.cxx

```diff
@@ -7,40 +7,43 @@
 
 /* Perftest Clock class */
 
- struct RTIClock *PerftestClock::Clock = RTIHighResolutionClock_new();
- struct RTINtpTime PerftestClock::ClockTime_aux = {0,0};
- RTI_UINT64 PerftestClock::Clock_sec = 0;
- RTI_UINT64 PerftestClock::Clock_usec = 0;
-
-void PerftestClock::Initialize() 
+PerftestClock::PerftestClock()
 {
-    if (PerftestClock::Clock == NULL) {
-        PerftestClock::Clock = RTIHighResolutionClock_new();
-    }
-    RTINtpTime_setZero(&PerftestClock::ClockTime_aux);
-    PerftestClock::Clock_sec = 0;
-    PerftestClock::Clock_usec = 0;
+    clock = RTIHighResolutionClock_new();
+    RTINtpTime_setZero(&clockTime_aux);
+
+    clock_sec = 0;
+    clock_usec = 0;
 }
 
-void PerftestClock::Finalize() 
+PerftestClock::~PerftestClock() 
 {
-    if (PerftestClock::Clock != NULL) {
-        RTIHighResolutionClock_delete(PerftestClock::Clock);
+    if (clock != NULL) {
```

Comment:
Check for null is unnecessary (as long as the constructor behaves correctly as pointed out in previous comment)

Reply:
Done.